r-package/grf/R/quantile_forest.R

```diff
@@ -44,6 +44,7 @@
 #'  Only applies if honesty is enabled. Default is TRUE.
 #' @param alpha A tuning parameter that controls the maximum imbalance of a split. Default is 0.05.
 #' @param imbalance.penalty A tuning parameter that controls how harshly imbalanced splits are penalized. Default is 0.
+#' @param compute.oob.predictions Whether OOB predictions on training set should be precomputed. Default is TRUE.
```

Comment:
I think we should default this to `FALSE`. My understanding is that quantile prediction is substantially more expensive than prediction with other forests, so it would be better precomputing OOB predictions were opt-in for users.

Reply:
I see, I just thought there was some room for breaking interface for a good cause, I have reverted these two changes and added a FALSE by default now.